Automattic/wp-calypso
# Components: Extract SiteMockup

This PR aims to pull out the base `SiteMockup` component for reuse elsewhere (There are plans to use this in the the importing signup flow, for instance), by taking out any state or actions and have the consuming component pass these as props.

#### Testing instructions

- Go to http://calypso.localhost:3000/start/onboarding-dev/site-style-with-preview
- Make sure the site preview renders properly as expected and with proper styling
- Following suite from the test plan of #30749, test that the click handler is still working as expected:
  - filter network events by `calypso_signup_site_preview_mockup_clicked`
  - Click anywhere on the site previews, making sure you see a tracking request in your network pane
